PowerShellTools/Intellisense/TabCompleteSession.cs

```diff
@@ -12,48 +12,56 @@ internal class TabCompleteSession
     {
         private IList<Completion> _completions;
         private int _index;
-        private bool _isInitialized;
+        private int _startPoint;
 
         /// <summary>
         /// Creates an uninitialized TabCompleteSession
         /// </summary>
         public TabCompleteSession()
         {
-            _isInitialized = false;
         }
 
         /// <summary>
         /// Creates and initializes the TabCompleteSession
         /// </summary>
-        /// <param name="activeSession">The active session</param>
-        public TabCompleteSession(ICompletionSession activeSession)
+        /// <param name="completions">The completions</param>
+        /// <param name="selectionStatus">The selection status</param>
+        /// <param name="startPoint">The start point of the completions</param>
+        public TabCompleteSession(IList<Completion> completions, CompletionSelectionStatus selectionStatus, int startPoint)
         {
-            Initialize(activeSession);
+            Initialize(completions, selectionStatus, startPoint);
         }
 
         /// <summary>
-        /// Returns true if the session has been initialized, otherwise false
+        /// Returns true if tab completion is enabled, otherwise false
         /// </summary>
-        public bool IsInitialized
+        public bool IsEnabled
         {
             get
             {
-                return _isInitialized;
+                return _completions != null && _completions.Count > 0;

```

Comment:
By looking at this, wondering is there any point to intialize(or even instantiate) tabcompletesession with an null/empty completion list?